docs/tutorial/quarkusgetstarted.adoc

```diff
@@ -15,217 +15,148 @@ This tutorial will walk through downloading and building one of the quarkus gett
+== Getting started
 
-== Getting started with Getting started
+We are going to use the link:https://quarkus.io/guides/getting-started[Quarkus getting started] project. The best practice is to open a terminal and test each command to get familiar with the input and output. We normally run commands in terminal before adding them to a qDup script to see how the commands behave.
 
-We are going to use the link:https://quarkus.io/guides/getting-started[Quarkus getting started] project to substitute for our application. We follow the steps in their guide the same way we would follow the README for other applications. The best practice is to open a terminal and test each command to get familiar with the output. Start with the first command in the README.
-
-```
-git clone https://github.com/quarkusio/quarkus-quickstarts.git
 ```
-The first README step clones the example repository. We create a qDup script with the same step, but our script will first cd to the `tmp` directory.
-```yaml
+#> cat <<EOF > quarkus.yaml
 scripts:
+  test-endpoint:
+  - wait-for: ready
+    then:
+    - sh: curl localhost:8080/hello
+    - signal: done #tells qDup that the "testing" is done
+  ensure-quarkus-cli:
+  - sh: quarkus version #to see if the command exists
+    then:
+    - regex: "command not found"
+      then: #install quarkus by following: https://quarkus.io/get-started/
+      - sh: "curl -Ls https://sh.jbang.dev | bash -s - trust add https://repo1.maven.org/maven2/io/quarkus/quarkus-cli/"
+      - sh: "curl -Ls https://sh.jbang.dev | bash -s - app install --fresh --force quarkus@quarkusio"
+      - sh: "source ~/.bashrc" # jbang change your environment if you don't have jbang in the PATH
   getting-started:
   - sh: cd /tmp/
-  - sh: git clone https://github.com/quarkusio/quarkus-quickstarts.git
+  - sh: "[[ ! -d quarkus-quickstarts ]] && git clone https://github.com/quarkusio/quarkus-quickstarts.git" # prevent cloning again
   - sh: cd quarkus-quickstarts
+  - sh: cd getting-started
+  - script: ensure-quarkus-cli
+  - sh:
+      command: quarkus dev
+      prompt:
+        "Press [e] to edit command line args (currently ''), [r] to resume testing, [o] Toggle test output, [:] for the terminal, [h] for more options>": "r"
+    timer:
+      2m: #a short timeout
+      - abort: quarkus dev did not exit before the timer expired
+    watch:
+    - regex: "Tests completed"
+      then:
+      - signal: ready
+    on-signal:
+      done:
+      - ctrlC #exits the process
 hosts:
-  test: me@localhost #replace this with your ssh username@hostname
+  test: $USER@localhost
 roles:
-  example:
+  setup-env:
     hosts:
     - test
     run-scripts:
     - getting-started
+    - test-endpoint
+EOF
 ```
 
-Create a `qdup.yaml` script file with the above yaml and run it using qDup
+Run with `java -jar qDup-0.8.5-uber.jar -i ~/.ssh/qdup quarkus.yaml -ix`. It will take some time and in the end you will see:
```

Comment:
Does `-ix` work after the list of yaml files? I though it had to occur before the list of yaml

Reply:
Adding `ix` after the YAML file is working